R/gadgets.R

```diff
@@ -0,0 +1,61 @@
+#' Open dataset selector
+#'
+#' @inheritParams listDatasets
+#' @return A `loadDataset()` call is pasted into your RStudio session`
+listDatasetGadget <- function(kind=c("active", "all", "archived"),
```

Comment:
Space between `function` and `(` please (ditto for all new functions in this PR).